jumpscale/packages/admin/actors/packages.py

```diff
@@ -26,8 +26,15 @@ def packages_names(self) -> str:
         return j.data.serializers.json.dumps({"data": list(self.threebot.packages.list_all())})
 
     @actor_method
-    def add_package(self, path: str = "", giturl: str = "") -> str:
-        return j.data.serializers.json.dumps({"data": self.threebot.packages.add(path=path, giturl=giturl)})
+    def add_package(self, path: str = "", giturl: str = "", extras: str = "") -> str:
+        kwargs = {}
+        if extras:
+            extras = extras.split(",")
+            for item in extras:
+                sub_items = item.split(":")
+                if sub_items and len(sub_items) > 1:
+                    kwargs[items[0].strip()]= sub_items[1].strip()
```

Comment:
items?

why is extra a string that needs to be processed like this and not just a dict?